app-frontend/src/app/components/colorCorrectScenes/colorCorrectScenes.scss

```diff
@@ -1,3 +1,4 @@
-rf-project-editor .leaflet-tile {
-  border: 1px solid black;
+rf-project-editor .grid-select .leaflet-tile {
```

Comment:
Add a TODO here for @designmatty so he can find it in case he wants to edit unless you've talked to him